fdbclient/ServerKnobs.h

```diff
@@ -412,6 +414,11 @@ class ServerKnobs : public KnobsImpl<ServerKnobs> {
 	                                             // CC_MAX_HEALTH_RECOVERY_COUNT within
 	                                             // CC_TRACKING_HEALTH_RECOVERY_INTERVAL.
 	int CC_MAX_HEALTH_RECOVERY_COUNT;
```

Comment:
Can you also comment what CC_MAX_HEALTH_RECOVERY_COUNT does? Also for the 2 new added knobs above.

Reply:
Done